how-to-build.md

```diff
@@ -18,30 +18,75 @@ each of our major platforms:
     sudo apt-get install build-essential
     sudo apt-get install libproc-dev
     sudo apt-get install libssl-dev
+    # To install cmake see Building CMake from Source below.
 
 ### Linux Mint 15, 16, Ubuntu 13.04, 13.10
 
     sudo apt-get install build-essential
     sudo apt-get install libprocps0-dev
     sudo apt-get install libssl-dev
+    # To install cmake see Building CMake from Source below.
```

Comment:
It is unlikely that these have a sufficiently new GCC by default. I say we drop these from the list.